WalletWasabi.Tests/UnitTests/Blockchain/TransactionBuilding/ChangelessTransactionCoinSelectorTests.cs

```diff
@@ -0,0 +1,58 @@
+using NBitcoin;
+using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Blockchain.TransactionBuilding;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Models;
+using WalletWasabi.Tests.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.Blockchain.TransactionBuilding;
+
+/// <summary>
+/// Tests for <see cref="ChangelessTransactionCoinSelector"/> class.
+/// </summary>
+public class ChangelessTransactionCoinSelectorTests
+{
+	[Fact]
+	public void FindBestSolution()
```

Comment:
This is the `PruneStrategy` test, right?

```suggestion
	public void FindBestSolutionWithPruneStrategy()
```

Reply:
Yes, but it is basically an implementation detail of `ChangelessTransactionCoinSelector` so it can also be `TryGetCoins`.